test/compilable/extra-files/json2.out

```diff
@@ -1,3 +1,4 @@
+--- ${RESULTS_DIR}/compilable/json2.out
```

Comment:
Why is this present here ?

Reply:
Because the header is written to `${RESULTS_DIR}/compilable/json2.out` instead of stdout, see `REQUIRED_ARGS` in `json2.d`

Reply:
Is this a blocker?